### ISSUE TITLE
Reorganize the Manual > Syntax page

### ISSUE THREAD

**Issue by mortenpi** [JuliaDocs/Documenter.jl#2246]
We should have a proper API reference page for the at-blocks, and then more tutorial/explainer content under the Manual section.

X-ref: https://github.com/JuliaDocs/Documenter.jl/pull/2168#discussion_r1251583387

**Comment by Raghavisdev** [comment]
I’m interested in working on this issue. Before starting, I wanted to clarify the intended structure.

Should the API reference for the `@` blocks be moved into a new page under the library section, while keeping the Syntax page focused on tutorial-style explanations?

**Comment by fingolfin** [comment]
No, that is the opposite of what the issue description suggests

**Comment by Raghavisdev** [comment]
Thanks for the clarification! @fingolfin 

So if I understand correctly, the goal would be to create a dedicated API reference page for the `@`-blocks (e.g. `@example`, `@repl`, `@docs`, etc.), and keep the Manual pages focused on explaining how and when to use them in practice.

Would the intention then be to extract the technical descriptions of these blocks from the current Syntax page into that API reference, while leaving higher-level explanations in the Manual?
